rskj-core/src/main/java/org/ethereum/net/server/ChannelManager.java

```diff
@@ -78,4 +78,6 @@
     Collection<Peer> getActivePeers();
 
     boolean isAddressBlockAvailable(InetAddress address);
+
+    Set<NodeID> broadcastTransactions(@Nonnull List<Transaction> transactions, @Nonnull Set<NodeID> nodeID);
```

Comment:
Be careful with the use of NodeID... most of the impl returns a copy of those. So, i wonder if it is actually cheaper to return the Node instead of NodeID.

Reply:
didn't get that. do you think it's worth it? this is just a reuse of broadcastTransaction which has been there before this PR